app/src/main/res/values/dimens.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
   <!-- Default screen margins, per the Android Design guidelines. -->
+  <dimen name="width_228dp">228dp</dimen>
+
+  <!-- Example dimension for height -->
+  <dimen name="height_88dp">88dp</dimen>
```

Comment:
This file is organized according to screen name, kindly add splash screen related dimens under the relevant category, or create one if it does not exist. Please stick to the patterns in the file.

Additionally, Your dimensions are in **dp**, contrary to the fix you proposed in https://github.com/oppia/oppia-android/issues/2652#issuecomment-1847579110.

Just adding the dependency and not using it does not fix the issue.